src/System.Drawing.Primitives/src/System/Drawing/Size.cs

```diff
@@ -190,5 +246,25 @@ public int Height
         ///    </para>
         /// </summary>
         public override string ToString() => "{Width=" + _width.ToString() + ", Height=" + _height.ToString() + "}";
+
+        /// <summary>
+        /// Helper method for operator * (multiply) that accepts type <see cref="Size"/> and <see cref="int"/>,
```

Comment:
I would just have a description of what the method actually does; you don't need to describe that it is a "helper" for another operator. The description can be essentially identical to the operator's description.